src/ifp/test/mock.tcl

```diff
@@ -0,0 +1,7 @@
+source "helpers.tcl"
+read_lef Nangate45/Nangate45.lef
+read_lef Nangate45/fake_macros.lef
+read_def mock.def
+initialize_floorplan -utilization 30 \
+  -aspect_ratio 0.5 \
+  -core_space 1 
```

Comment:
The test needs to do something to validate the results.  Likely you want a write_def and a .defok file.  Please see other tests for examples.

Reply:
yes, absolutely. I just wanted to verify that the lef/def files and the mock macros are OK and this is indeed testing a double height cell before spending time on a defok file.

Should I proceed with it?

Reply:
yes